lib/openstudio-standards/btap/bridging.rb

```diff
@@ -734,6 +732,17 @@ def initialize(model = nil, argh = {})
           next unless v[:stypes] == stypes
 
           v[:r] = TBD.resetUo(lc, v[:filmRSI], v[:index], v[:uo])
+
+          # Maintain initial uprated Uo as AdditionalProperty.
+          v[:surfaces].each do |id|
+            surface = model.getSurfaceByName(id)
+            next if surface.empty?
+
+            surface = surface.get
+            next unless surface.additionalProperties.getFeatureAsDouble(tag).empty?
+
+            surface.additionalProperties.setFeature(tag, v[:uo])
```

Comment:
Maintaining surface-specific, _uprated_ Uo factors, for post-simulation costing and embodied carbon calculations.